src/huggingface_hub/hf_api.py

```diff
@@ -50,6 +50,28 @@
 USERNAME_PLACEHOLDER = "hf_user"
 
 
+# TODO: remove after deprecation period is over (v0.7)
+def _validate_repo_id_deprecation(repo_id, name, organization):
+    """Returns (name, organization) from the input."""
+    if repo_id and (name or organization):
+        raise ValueError(
+            "Only pass `repo_id` and leave deprecated `name` and "
+            "`organization` to be None."
+        )
+    elif name or organization:
+        warnings.warn(
+            "`name` and `organization` input arguments are deprecated and "
+            "will be removed in v0.7. Pass `repo_id` instead.",
+            FutureWarning,
+        )
+    else:
+        if "/" in repo_id:
+            name, organization = repo_id.split("/")
+        else:
+            name, organization = repo_id, None
```

Comment:
Current CI failure comes from here, the `name` and `organization` are switched: we're expecting `<org>/<repo_id>` rather than the other way around.

```suggestion
        if "/" in repo_id:
            organization, name = repo_id.split("/")
        else:
            organization, name = None, repo_id
```

Reply:
Ideally this method would be unit-tested so as to raise an error from the method itself rather than for another test to raise a seemingly unrelated error